extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -373,7 +382,9 @@ export abstract class AzureAuth implements vscode.Disposable {
 			throw new AzureAuthError(msg, 'Adding account to cache failed', undefined);
 		}
 		try {
+			Logger.pii(`Saving access token`, [{ name: 'access', objOrArray: accessToken }], []);
```

Comment:
What's this empty array for? And why the object? Probably easier just to have the string with the embedded stuff in it directly

Reply:
This is because the strings to shorten is passed in as an array but here we don't have anything to shorten